SPO/README.md

```diff
@@ -5,8 +5,12 @@ Please find the instructions [here](https://docs.microsoft.com/en-us/azure/azure
 
 ## Set up Application Insights 
 1. Follow [SPFx Extensions instruction](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/get-started/build-a-hello-world-extension) to create a SharePoint Framework Extension.
-2. Install Application Insights via [NPM setup](https://github.com/microsoft/ApplicationInsights-JS#getting-started). 
-3. Under your Extension solution src folder, update `ApplicationCustomizer.ts` file:
+2. Add Application Insights to your extension solution. **There are 2 ways to do this.**
```

Comment:
In our official docs, we add a Note about why you would use each method. 
https://docs.microsoft.com/en-us/azure/azure-monitor/app/javascript#adding-the-javascript-sdk
Does this apply to SPFx too? Are there any differences?

Reply:
I think one difference is we can't control when extension gets loaded in SP site. There are many scripts loaded before our SDK like this: 
![image](https://user-images.githubusercontent.com/20292261/91897699-85d32400-ec4f-11ea-829f-515e19156ec8.png)
The second thing I can think of is since SPFx extension is an extra step for users to setup AppInsights for SP, if they choose snippet, they don't need to update the AI dependency versions and [update the extension solution](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/get-started/hosting-extension-from-office365-cdn#update-your-solution-project-for-the-cdn-urls) bundle every time our SDK updates.

I can't think of other differences, @MSNev do you have more to add? 😬

Reply:
@xiao-lix So basically it's this?
NPM = more stability
Snippet = latest features